textacy/fileio/write.py

```diff
@@ -3,31 +3,56 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import gzip
-import io
 import json
+
 from numpy import savez, savez_compressed
-import os
 from scipy.sparse import csc_matrix, csr_matrix
-
 from spacy.tokens.doc import Doc as SpacyDoc
 
-from textacy.compat import bzip_open
+from textacy.compat import (bytes_to_unicode, bytes_type,
+                            unicode_to_bytes, unicode_type)
+from textacy.fileio import open_sesame, make_dirs
 
 
-def _make_dirs(filename):
-    head, _ = os.path.split(filename)
-    if head and not os.path.exists(head):
-        os.makedirs(head)
+# TODO: keep this, or no?
```

Comment:
Not currently, no. I might add it in as an option, though, or just leave it as a non-private function.